python/rapidsmpf/rapidsmpf/shuffler.pyx

```diff
@@ -136,9 +139,16 @@ cdef class Shuffler:
         -----
         This method adds the given chunks to the shuffle, associating them with their
         respective partition IDs.
+
+        There are some considerations for using ``grouped=True``:
```

Comment:
I suspect this portion of the docstring isn't needed?

May be worth adding a `See Also` section This would be roughly:

```
See Also
--------
Shuffler.concat_insert
```

and likewise for `Shuffler.concat_insert` to refer to `Shuffler.insert_chunks`.